st.c

```diff
@@ -4067,6 +4078,7 @@ kbdkey(void *data, struct wl_keyboard *keyboard, uint32_t serial, uint32_t time,
 		return;
 
 	if (state == WL_KEYBOARD_KEY_STATE_RELEASED) {
+		wl.globalserial = serial;
```

Comment:
Shouldn't the serial be saved on key press, not key release?

The key press is the action that is likely to trigger tmux to send the set selection escape sequence, right?